base_tier_validation_waiting/models/tier_validation.py

```diff
@@ -51,3 +56,11 @@ def _validate_tier(self, tiers=False):
         self._validate_tier_waiting_reviews(tiers)
         res = super(TierValidation, self)._validate_tier(tiers)
         return res
+
+    def _compute_validation_status(self):
+        res = super()._compute_validation_status()
+        for item in self:
```

Comment:
The waiting state should only be applied if not approved, rejected, pending.

```suggestion
        for item in self.filtered(lambda tv: tv.validation_status == "no"):
```